src/main/java/core/basesyntax/model/Transaction.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.model;
+
+public class Transaction {
+    private final Operation operation;
+    private final String productName;
+    private final int quantity;
+
+    public Transaction(String operation, String productName, int quantity) {
```

Comment:
```suggestion
    public Transaction(Operation operation, String productName, int quantity) {
```
Constructor is not a good place for business logic (like converting String to Operation). Getting exceptions from the constructor is usually unexpected.